src/pages/index.js

```diff
@@ -263,10 +263,7 @@ class BlogRoll extends React.Component {
     {
       url: "https%3A%2F%2Fpchaigno.github.io%2Ffeed.xml",
       author: "Paul Chaignon",
-      filterBy: (post) =>
-        post.categories.some((category) =>
-          category.toLowerCase().includes("bpf")
-        ),
+      filterBy: (post) => post.title.toLowerCase().includes("bpf"),
```

Comment:
It looks like this PR reverted https://github.com/cilium/ebpf.io/pull/72 by mistake. I'll resend it.